src/variables.scss

```diff
@@ -5,6 +5,8 @@ $neutral: var(--qr-color-neutral, hsl(0, 0%, 45%));
 $neutral-light: var(--qr-color-neutral-light, hsl(0, 0%, 96%));
 $neutral-dark: var(--qr-color-neutral-dark, hsl(0, 0%, 32%));
 
+$border-neutral: hsl(from $neutral-light h s 80%);
```

Comment:
I could not use `calc` here. I did not figure out why yet. I hope the `80%` for the lightness gets the job done.